lib/helpers/get-paths-sync.js

```diff
@@ -18,8 +18,12 @@ module.exports = function getPathsSync(patterns, config) {
     return patterns;
   }
 
+  const isWindows = process.platform === 'win32';
   //Prepare glob config
-  const cfg = Object.assign({ignore}, globConfig, {nodir: true});
+  const cfg = Object.assign({ignore}, globConfig, {
+    nodir: true,
+    windowsPathsNoEscape: isWindows
```

Comment:
Since the flag already is called `windowsPathsNoEscape`, my guess is that it explicitly works only in windows.
Is it then safe to just set this flag to `true`, and we don't need to check `isWindows` first?

Reply:
> Glob patterns should always use / as a path separator, even on Windows systems, as \ is used to escape glob characters. If you wish to use \ as a path separator instead of using it as an escape character on Windows platforms, you may set windowsPathsNoEscape:true in the options. In this mode, special glob characters cannot be escaped, making it impossible to match a literal * ? and so on in filenames.

I reckon we may have to set this behind a flag, otherwise it will potentially break functionality for people

Reply:
> Since the flag already is called windowsPathsNoEscape, my guess is that it explicitly works only in windows.

Your guess is as good as mine, but if the option is not windows specific, you risk introducing a breaking change where escape character (`\`) no longer works on linux.

> I reckon we may have to set this behind a flag, otherwise it will potentially break functionality for people

This is not a new feature. This is a fix for a regression introduced in `v7`.
See the following issues for more info:
https://github.com/ngneat/transloco/issues/700
https://github.com/adamreisnz/replace-in-file/issues/165

Reply:
Checked [the `glob` source code](https://github.com/isaacs/node-glob/blob/97611cd366e5906a4c58df3f5214af843b0a5e63/src/glob.ts#L424): if you remove OS check, you will break escape character on linux.